1/contrib/jenkins/kube-slave-common.sh

```diff
@@ -121,6 +122,7 @@ function generate_kubernetes_config() {
           <envVars/>
           <nodeSelector/>
           <remoteFs>/tmp</remoteFs>
+          <serviceAccount>${oc_serviceaccount_name}</serviceAccount>
```

Comment:
@bparees It is called here https://github.com/tnozicka/jenkins/blob/master/1/contrib/jenkins/kube-slave-common.sh#L102 .  I am not sure why github search failed there...

I will add it there as well.